<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录页面</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{border-image: url(:/Pictures/login.jpg);}

QLineEdit {
	font: 12pt &quot;Microsoft YaHei UI&quot;; 
	background: rgba(255,255,255,0.08);
    border-color: rgba(58,123,255,0.3);
}

QLineEdit:hover {
	font: 12pt &quot;Microsoft YaHei UI&quot;;
	background: rgba(255,255,255,1);
    border-color: rgba(58,123,255,1);
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #F8FAFF, stop:1 #EFF2F7);
    border: 1px solid rgba(58,123,255,0.15);
    border-radius: 6px;
    color: #3A7BFF;
    font: 500 22px &quot;微软雅黑&quot;;
    padding: 10px 24px;
    min-width: 92px;
}

QPushButton:hover {
    background: rgba(58,123,255,0.08);
    border-color: rgba(58,123,255,0.3);
}

QPushButton:pressed {
    background: rgba(58,123,255,0.15);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,2">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="6">
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="4">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLineEdit" name="li_account">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>请输入账号</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLineEdit" name="li_password">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
              <property name="placeholderText">
               <string>请输入密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_show_p">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>显示
密码</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="leftMargin">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_log">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_regist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::StrongFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
